javascripts/discourse/connectors/above-topic-list-item/clickable-topic-row.js

```diff
@@ -0,0 +1,43 @@
+import Component from "@glimmer/component";
+import { action } from "@ember/object";
+import { wantsNewWindow } from "discourse/lib/intercept-click";
+import { navigateToTopic } from "discourse/components/topic-list-item";
+import { bind } from "discourse-common/utils/decorators";
+import { inject as service } from "@ember/service";
+
+export default class extends Component {
+  @service site;
+
+  @bind
+  clickHandler(event) {
+    const targetElement = event.target;
+    const topic = this.args.outletArgs.topic;
+
+    const clickTargets = [
+      "topic-list-data",
+      "link-bottom-line",
+      "topic-list-item",
+    ];
+
+    if (this.site.mobileView) {
+      clickTargets.push("right", "clearfix");
```

Comment:
I find these two classes a bit too generic for targeting and I am wondering if we should add in core a more descriptive class to the element we want to target for the mobile view, like we have for desktop.